cli/util/util.go

```diff
@@ -938,3 +940,31 @@ func CopyFileDeep(src, dst string) error {
 	}
 	return copy.Copy(src, dst)
 }
+
+// StringToTimestamp transforms string with number or RFC339Nano time
+// to <sec.nanosec> timestamp string.
+func StringToTimestamp(input string) (string, error) {
+	if input == "" {
+		// Default value.
+		return strconv.FormatUint(math.MaxUint64, 10), nil
+	}
+
+	// Float.
+	floatTimestamp, err := strconv.ParseFloat(input, 64)
+	if err == nil {
+		return strconv.FormatFloat(floatTimestamp, 'f', -1, 64), nil
+	}
+
+	// RFC3339/RFC3339Nano = "2006-01-02T15:04:05.999999999Z07:00"
+	rfc3339NanoTs, err := time.Parse(time.RFC3339Nano, string(input))
+	if err == nil {
+		tsSec := rfc3339NanoTs.Unix()
+		tsNanoSec := rfc3339NanoTs.Nanosecond()
+		ts := fmt.Sprintf("%s.%s", strconv.FormatInt(tsSec, 10),
+			strconv.FormatInt(int64(tsNanoSec), 10))
+		return ts, nil
+	}
+
+	// Incorrect input, trigger an error.
+	return "", err
```

Comment:
To stick to the Go style guide. I think that it is easier to read in this way:

```suggestion
	rfc3339NanoTs, err := time.Parse(time.RFC3339Nano, string(input))
	if err != nil {
		// Incorrect input, trigger an error.
	        return "", err
	}

	tsSec := rfc3339NanoTs.Unix()
	tsNanoSec := rfc3339NanoTs.Nanosecond()
	ts := fmt.Sprintf("%s.%s", strconv.FormatInt(tsSec, 10),
		strconv.FormatInt(int64(tsNanoSec), 10))
	
	return ts, nil
```
```

Reply:
[From my point of view](https://github.com/tarantool/tt/pull/1026#discussion_r1848020610) it easier to see 3 positive results and after that 1 negative, than their mix.
But ok, has been changed.

Reply:
It's more about Go codestyle, when we handle an error in separate block after function call and positive cases just go after it.